proposals/0000-accept-prs-on-github.rst

```diff
@@ -0,0 +1,193 @@
+.. proposal-number:: Leave blank. This will be filled in when the proposal is
+                     accepted.
+
+.. trac-ticket:: Leave blank. This will eventually be filled with the Trac
+                 ticket number which will track the progress of the
+                 implementation of the feature.
+
+.. implemented:: Leave blank. This will be filled in with the first GHC version which
+                 implements the described feature.
+
+.. highlight:: haskell
+
+Accept PRs on GitHub
+====================
+
+This proposes to accept pull requests (PRs) against GHC on GitHub (GH). There are a few
+possible mechanisms for doing so, as delineated below.
+
+This proposal comes directly out of a `conversation <https://mail.haskell.org/pipermail/ghc-devs/2016-September/012828.html>`_
+on the mailing lists.
+
+Motivation
+----------
+
+The overhead involved in getting started with the current Phabricator
+based process for submitting patches for review and merging them is
+non-trivial. One needs to,
+
+* setup an account on Phabricator,
+* setup an account on Trac,
```

Comment:
Wouldn't this be needed in exactly the same circumstances regardless of whether the patch is submitted via github or Phab?  Similarly for "Update the Trac ticket".  It's entirely possible to `arc diff` without having a Trac account or a ticket.